test-scenarios/package.json

```diff
@@ -36,6 +36,7 @@
     "ember-cli-typescript-2": "npm:ember-cli-typescript@^2.0.0-beta.3",
     "ember-cli-typescript-3": "npm:ember-cli-typescript@^3.0.0",
     "ember-cli-typescript-4": "npm:ember-cli-typescript@^4.0.0-alpha.1",
+    "ember-engines": "^0.8.20",
```

Comment:
now should be `"ember-engines": "^0.9.0"` as v0.9.0 what includes support for ember-asset-loader@1.0.0